### PR TITLE
feat(readMe): updated with new verbiage provided by Yolanda

### PR DESCRIPTION
**Pull request by sydbails**
## Description
This addresses the Read Me page with to update the verbiage with new text provided by @yrabun and approved by @drealuc 
You can see [the document here](https://ibm.ent.box.com/file/733400051445?s=c9f9x3oout8tz6t8l8n052qyhn73y1i3&utm_campaign=collab%20auto%20accept%20user&utm_medium=email&utm_source=trans).

Please reach out to @yrabun or @drealuc if you need access to this file. 

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Locally in dev env

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings

Signed-off-by: Syd <sydabailey@gmail.com>

### CODE REVIEW

**Review by davidnixon — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -6,35 +6,35 @@ This solution starter was created by technologists from IBM.
...
+- **Chris Stefano** _Generalist_
+- **Ricky Ellison** _Generalist_
+- **Mike Boone** _Sr. Security & IP Advisor_
 
 # What is the problem?
 
-For centuries, voters in disparate communities, particularly Black voters, have experienced obstacles in casting their votes be it situational or intentionally constructed. Voting has been perceived an intangible and unfair process based on restrictive local processes, changing requirements, regulations, inability to access the correct voting location, and lack of information. As a result millions of minority votes go uncounted, purged, or simply not submitted do to lack of faith and trust in the system.
+American History has shown that voter suppression has led to corruption and a weak democracy that does not reflect the will of the people.Voting has been perceived an intangible and unfair process based on restrictive local processes, changing requirements, regulations, an inability to access the correct voting location, and a lackof information. As a result, millions of minorities, especially Black people, have had their votes go uncounted, purged, or simply not submitted due to lack of faith and trust in the system.
```
Comment (GeraldMit) [inline README.md:29]: - History -> history
- lackof -> lack of
- people.Voting -> people. Voting 
- (suggestion for simplicity) submitted due to lack of [faith and trust] in the system -> submitted due to a lack of trust in the system.

File: README.md [hunk 2]
```diff
@@ -6,35 +6,35 @@ This solution starter was created by technologists from IBM.
...
+The **Five Fifths Voter** solution starter focuses on addressing thekey areas of voter suppression such as voter registration, voter ID laws, voter registration restrictions, voter purging, felony disenfranchisement, and gerrymandering all of which are contributors to disenfranchising minorities and people with disabilities.The solutionhelps with these challenges by providing a capability to help ensure disenfranchised minority voters are educated, empowered, and enabled to overcome setbacks incurred by voter suppression. Five Fifths Voter helps to identify the correct polling stations, notifies users of requirements needed for their state, and personal circumstances as it pertains to different types of voting, deadlines, and quick accessto information.
```
Comment (GeraldMit) [inline README.md:31]: thekey -> the key
solutionhelps -> solution  helps  -> (suggestion replace "solution helps with" to be more dynamic)  confronts | counteracts | eases | surmounts... )
solutionhelps ... capability to help ensure  -> too many "helps" see suggestion above.
accessto -> access to

File: README.md [hunk 3]
```diff
@@ -6,35 +6,35 @@ This solution starter was created by technologists from IBM.
...
 
-Furthermore, **Five Fifths Voter** strives to evolve the culture of voting and enable users to build a relationship with us through having the ability to select personalized features that enhance their experience and interact with other users to become a part of a community.
```
Comment (GeraldMit) [inline README.md:34]: through having the ability to -> through the ability to

**Review by GeraldMit — APPROVED** [review]
Approved. I noticed some minor spacing and wording I have commented inline for correctiion.

**Pull request merged**